chromium_src/chrome/browser/ui/webui/settings/metrics_reporting_handler.h

```diff
@@ -6,14 +6,8 @@
 #ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_WEBUI_SETTINGS_METRICS_REPORTING_HANDLER_H_
 #define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_UI_WEBUI_SETTINGS_METRICS_REPORTING_HANDLER_H_
 
-#if defined(BRAVE_CHROMIUM_BUILD)
 #define GOOGLE_CHROME_BUILD
```

Comment:
Same, unrelated to this PR, but we should probably include all headers from the original file here.

Reply:
Ditto (i.e. will do on a separate PR)

Reply:
As clarified above, the change is small enough (and only affects these files) that I'm including it in this PR in the end.